### PR TITLE
lint pass 2

### PR DESCRIPTION
**Pull request by aus-social**
Learning Ruby by fixing code quality alerts... as per discussion with @nightpool, some of these things are by choice.. please feel free to veto these or comment away

### CODE REVIEW

File: config/initializers/omniauth.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 end
 
 Devise.setup do |config|
-  # Devise omniauth strategies
+  #  Devise omniauth strategies
```
Comment (Gargron) [inline config/initializers/omniauth.rb]: This is strange

File: spec/controllers/settings/applications_controller_spec.rb [hunk 2]
```diff
@@ -120,9 +120,9 @@ def call_create
 
       def call_update
         patch :update, params: {
-                id: app.id,
+          id: app.id,
                 doorkeeper_application: opts
```
Comment (Gargron) [inline spec/controllers/settings/applications_controller_spec.rb]: I don't think this is right

File: config/initializers/omniauth.rb (hunk 1 above)
Comment (ashleyhull-versent) [reply config/initializers/omniauth.rb]: correct... I think this is a bashism..

**Review by shleeable — CHANGES_REQUESTED** [review]

File: spec/controllers/settings/applications_controller_spec.rb (hunk 2 above)
Comment (ashleyhull-versent) [reply spec/controllers/settings/applications_controller_spec.rb]: I missed the second line?

File: spec/controllers/settings/applications_controller_spec.rb [hunk 3]
```diff
@@ -139,14 +139,14 @@ def call_update
     context 'failure' do
       before do
         patch :update, params: {
-                id: app.id,
+          id: app.id,
                 doorkeeper_application: {
```
Comment (Gargron) [inline spec/controllers/settings/applications_controller_spec.rb]: Spacing off here as well

Comment (aus-social) [reply spec/controllers/settings/applications_controller_spec.rb]: I re-alligned the whole block -  I think this is better now.

**Review by Gargron — APPROVED** [review]

**Pull request merged**